src/ttt/ui.clj

```diff
@@ -0,0 +1,31 @@
+(ns ttt.ui
```

Comment:
I'm not as familiar w/ how Clojure works.  Are functions like `print-board` available even without the namespace reference?  I was thinking (and may be incorrect) that it would look like this


```clojure
(ns ttt.ui
  (:require ...)
  (defn print-board [] ...))
```

in other words - the functions would be inside the namespace declaration?  I could well be wrong; it's just not as common for LISP-styled languages to use side-effects like that.  A quick experiment of trying to call `print-board` without its namespace prefix would verify how it's working.


...or I could break down and actually read something about how Clojure really works.

Reply:
When i need to use print-board in another file, like in core.clj, then i call it using the namespace. there i would use `ui/print-board`. 🤷🏻‍♀️